### DISCUSSION TITLE
What happens when neither parallel nor piped are enabled?

### DISCUSSION THREAD

**Discussion by silvenon** [evilmartians/lefthook#723] [Q&A] [answered]
This isn't explicitly documented. When not enabling `parallel` or `piped`, do commands still run in sequence, but unlike `piped` the failed one is skipped and it proceeds running the next one? And if they do run in sequence, is that sequence also alphabetical and can it be altered with `priority`?

**Comment by mrexox** [comment, marked as answer]
When not enabling `parallel` or `piped` the commands still run sequentially. The default order is alphanumeric and it can be changed with adding `priority`. I hope that helps :)

The difference between `piped` and the default behavior is that with `piped` the hook will stop on the first error, so next commands won't be executed.

**Reply by silvenon** [reply]
Thanks! Although omitting `parallel` implies sequential, it would be nice to update the documentation to not tie the ordering stuff to `piped`. I would like to help if contributions to docs are welcome.

**Reply by silvenon** [reply]
And the fact that my guess matched the behavior means that the API is good ⭐

**Reply by mrexox** [reply]
Of course, contributions to docs are welcome. I'd be glad if you submitted a PR 👍
